src/ontology/imports/general_import.ofn

```diff
@@ -3659,7 +3659,6 @@ Declaration(AnnotationProperty(oboInOwl:hasOBONamespace))
 Declaration(AnnotationProperty(oboInOwl:hasRelatedSynonym))
 Declaration(AnnotationProperty(oboInOwl:id))
 Declaration(AnnotationProperty(oboInOwl:inSubset))
-Declaration(AnnotationProperty(rdf:type))
```

Comment:
This particular thing will pop up again - its coming in via an OntoFox import that is fairly frequently regenerated.  Will that be a problem?

Reply:
Well, it is  not legal OWL and now with the new ROBOT will crash all our QC pipelines depending on FOODON moving forward - so I guess you will either have to fix the OntoFox import system, or add a ROBOT command to remove this after the fact? Right now, we have to create custom FOODON imports for MAXO, Mondo and all other ontologies dependent of FOODON just to remove that axiom..

Reply:
Ok, weve implemented your additional checks.  Seems to be running well, and no owl2dl issues.

Reply:
Great!